### ISSUE TITLE
error[E0432]: unresolved imports  wont compile

### ISSUE THREAD

**Issue by oneforall2** [koverstreet/bcachefs-tools#338]
Compiling bch_bindgen v0.1.0 (/tmp/bcachefs-tools-1.11.0/bch_bindgen)
error[E0432]: unresolved imports `bch_bindgen::c::BCH_IOCTL_SUBVOLUME_CREATE`, `bch_bindgen::c::BCH_IOCTL_SUBVOLUME_DESTROY`
 --> src/wrappers/handle.rs:4:73
  |
4 |     bcache_fs_close, bcache_fs_open, bch_ioctl_subvolume, bchfs_handle, BCH_IOCTL_SUBVOLUME_CREATE,
  |                                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `BCH_IOCTL_SUBVOLUME_CREATE` in `c`
5 |     BCH_IOCTL_SUBVOLUME_DESTROY, BCH_SUBVOL_SNAPSHOT_CREATE,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `BCH_IOCTL_SUBVOLUME_DESTROY` in `c`

warning: unexpected `cfg` condition name: `fuse`
  --> src/bcachefs.rs:66:19
   |
66 |             #[cfg(fuse)]
   |                   ^^^^
   |
   = help: expected names are: `clippy`, `debug_assertions`, `doc`, `docsrs`, `doctest`, `feature`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
   = help: consider using a Cargo feature instead

**Comment by oneforall2** [comment]
[](https://pastebin.com/Ne4HVwp2)

**Comment by gdevenyi** [comment]
Same error, fixed formatting:
```
   Compiling bch_bindgen v0.1.0 (/home/gdevenyi/system/bcachefs-tools/bch_bindgen)
error[E0432]: unresolved imports `bch_bindgen::c::BCH_IOCTL_SUBVOLUME_CREATE`, `bch_bindgen::c::BCH_IOCTL_SUBVOLUME_DESTROY`
 --> src/wrappers/handle.rs:4:73
  |
4 |     bcache_fs_close, bcache_fs_open, bch_ioctl_subvolume, bchfs_handle, BCH_IOCTL_SUBVOLUME_CREATE,
  |                                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `BCH_IOCTL_SUBVOLUME_CREATE` in `c`
5 |     BCH_IOCTL_SUBVOLUME_DESTROY, BCH_SUBVOL_SNAPSHOT_CREATE,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `BCH_IOCTL_SUBVOLUME_DESTROY` in `c`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `bcachefs-tools` (bin "bcachefs") due to 1 previous error
make: *** [Makefile:176: bcachefs] Error 101
```

**Comment by koverstreet** [comment]
I just updated the cargo dependencies (this was caused by a bindgen issue that's been fixed), and tagged 1.13.0.
